scripts/jsdeps.py

```diff
@@ -0,0 +1,39 @@
+import json
+import shutil
+from pathlib import Path
+
+repoRoot = Path(__file__).resolve().parent.parent
+destDir = repoRoot / "src/fontra/client/third-party/"
+
+
+def loadPackageDependencies():
+    with open("package.json") as packageFile:
+        packageInfo = json.load(packageFile)
+        dependencies = packageInfo.get("dependencies")
+        if dependencies:
+            return list(dependencies.keys())
+    return []
+
+
+def processDependencies(dependencies):
+    if destDir.is_dir():
+        shutil.rmtree(destDir)
+    for dependency in dependencies:
+        processDependency(name=dependency)
+
+
+def processDependency(name):
+    src = Path("node_modules") / name
```

Comment:
You could add another global const:
`nodeModules = repoRoot /  "node_modules"`
This ensures it will work correctly regardless of the current working dir.